src/test/java/org/cactoos/ScalarTest.java

```diff
@@ -35,24 +35,28 @@
  */
 public final class ScalarTest {
 
-    /**
-     * A rule for handling an exception.
-     */
-    @Rule
-    public final ExpectedException cause = ExpectedException.none();
-
     @Test
-    public void failForNullArgument() throws Exception {
-        this.cause.expect(IllegalArgumentException.class);
-        this.cause.expectMessage("NULL instead of a valid scalar");
-        new NoNulls<>(null).value();
+    public void failForNullArgument() {
+        MatcherAssert.assertThat(
```

Comment:
@atapin please use `Assertion` instead of the static methods from `MatcherAssert`